capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -8,10 +8,12 @@
  * JSON data directly from the CollegeScorecard API.
  */
 @AutoValue
-public abstract class ScorecardData {
+private abstract class ScorecardData {
```

Comment:
Everything (including this class) should be package private. See [this access control documentation](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). If we make things private then they won't be usable by other classes the scorecard package.

Reply:
i dont think autovalue doesnt support this. the documentation says autovalue methods at least needs to be visible by the package, so i can remove ```public``` but i cant add private, and i think it applies to the class as well because when i add ```private``` here, it doesnt build.

Reply:
Right - I'm suggesting that this class should be `package private` (without a modifier)